sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayContributionRole.scala

```diff
@@ -0,0 +1,17 @@
+package uk.ac.wellcome.display.models.v2
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+
+@ApiModel(
+  value = "ContributionRole",
+  description =
+    "A contribution role"
```

Comment:
I know this is probably not the correct documentation line we want to show. I couldn't find the schema for the contributors model so I'm going to chase with @silveroliver on that. For the time being, I'm putting in some deliberately basic descriptions